src/olympia/editors/templates/editors/reviewlog.html

```diff
@@ -43,9 +43,15 @@
                   {% if item.arguments|count >= 2 %}
                     {{ item.arguments[1] }}
                   {% endif %}
+                  {% if item.arguments[1].channel == amo.RELEASE_CHANNEL_LISTED %}
                   <a href="{{ url('editors.review', item.arguments[0].slug) }}">
                     {{ ACTION_DICT.get(item.action) }}
                   </a>
+                  {% else %}
+                  <a href="{{ url('editors.review', 'unlisted', item.arguments[0].slug) }}">
+                    {{ ACTION_DICT.get(item.action) }}
+                  </a>
+                  {% endif %}
```

Comment:
I've been thinking about the (rare) cases the review log doesn't have a version.  We don't want those to fail.  And also it'd be better to not have the <a... > twice.  So lets do:
```
                  {% if item.arguments|count >= 2 %}
                    {{ item.arguments[1] }}
                    {% set channel = item.arguments[1].channel %}
                  {% else %}
                    {% set channel = 0 %}
                  {% endif %}
                  {% if channel == amo.RELEASE_CHANNEL_UNLISTED %}
                    {% set review_url = url('editors.review', 'unlisted', item.arguments[0].slug) %}
                  {% else %}
                    {% set review_url = url('editors.review', item.arguments[0].slug) %}
                  {% endif %}
                  <a href="{{ review_url }}">
                    {{ ACTION_DICT.get(item.action) }}
                  </a>
```